pt_miniscreen/hotspots/text_hotspot.py

```diff
@@ -57,3 +59,14 @@ def text(self):
     @text.setter
     def text(self, value_or_callback):
         self._text = value_or_callback
+
+    @property
+    def text_size(self):
```

Comment:
So if we're using this to determine the size of the text in a text hotspot, then we can use this to adjust the `xy` of the overall hotspot to ensure that the centre of the text maintains its position. However, I'm not sure that this is actually being done, given the fact that only the battery page actually makes use of this, and I'm not sure that the `xy` position is getting updated on text changes.

Can you confirm?